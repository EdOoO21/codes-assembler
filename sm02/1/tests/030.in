654 -408 251 -939 273 338 693 -570 1000 -474 -901 -187 -230 313 -726 -834 -55 -985 69 909 -504 -948 -852 -673 860 724 671 227 87 878 -178 991 703 340 -287 96 806 -862 -178 -952 -491 649 152 376 -427 562 -122 158 -805 380 -430 308 151 833 986 -772 256 12 912 -367 -736 698 -368 796 -362 -484 38 -836 -717
