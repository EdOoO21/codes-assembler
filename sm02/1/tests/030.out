-717
-836
38
-484
-362
796
-368
698
-736
-367
912
12
256
-772
986
833
151
308
-430
380
-805
158
-122
562
-427
376
152
649
-491
-952
-178
-862
806
96
-287
340
703
991
-178
878
87
227
671
724
860
-673
-852
-948
-504
909
69
-985
-55
-834
-726
313
-230
-187
-901
-474
1000
-570
693
338
273
-939
251
-408
654
